### PR TITLE
Feature/request rename

### PR DESCRIPTION
**Pull request by wmcginty**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
     
-* Added a new initalizer to `AnyNetworkRequest` which accepts a `String` value designating the key of JSON at which to begin decoding.
+* Added a new initalizer to `AnyRequest` which accepts a `String` value designating the key of JSON at which to begin decoding.
     [Will McGinty](https://github.com/wmcginty)
     [#41](https://github.com/BottleRocketStudios/iOS-Hyperspace/pull/41)
     
-* Separated the generation/encoding of the URL query from the `NetworkRequest` object into an extension `URL`.
+* Separated the generation/encoding of the URL query from the `Request` object into an extension `URL`.
     [Will McGinty](https://github.com/wmcginty)
     [#40](https://github.com/BottleRocketStudios/iOS-Hyperspace/pull/40)
```
Comment (tylermilner) [inline CHANGELOG.md]: I think you're just missing a new entry in here for the actual renaming. 😉

File: README.md [hunk 2]
```diff
@@ -61,10 +61,10 @@ struct CreatePostRequest: NetworkRequest {
 }
 ```
 
-#### Option 2 - Using the `AnyNetworkRequest<T>` struct
+#### Option 2 - Using the `AnyRequest<T>` struct
 
 ```swift
-let createPostRequest = AnyNetworkRequest<Post>(method: .post,
+let createPostRequest = AnyRequest<Post>(method: .post,
                                                 url: URL(string: "http://jsonplaceholder.typicode.com/posts")!,
```
Comment (tylermilner) [inline README.md]: For readability, let's get these parameters lined up.

File: CHANGELOG.md (hunk 1 above)
Comment (wmcginty) [reply CHANGELOG.md]: Fixed

File: README.md (hunk 2 above)
Comment (wmcginty) [reply README.md]: Fixed

**Pull request merged**